lib/helpers.coffee

```diff
@@ -31,7 +31,9 @@ module.exports = Helpers =
         else
           resolve(data.stderr.join(''))
       if isNodeExecutable
-        spawnedProcess = new BufferedNodeProcess({command, args, stdout, stderr, exit})
+        options.env = JSON.parse(JSON.stringify(process.env))
```

Comment:
Wow can't believe I didn't know that... So I looked more into Object.create and an interesting thing about it is that the properties created are not writable, enumerable or configurable by default